crates/tendermint/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name        = "tendermint"
```

Comment:
I played around with a few different names for this crate before deciding that `tendermint` was probably a good enough name for it.